src/common/mem.c

```diff
@@ -1136,10 +1237,15 @@ set_offsets(struct umem_instance *umm)
 	}
 
 #ifdef DAOS_PMEM_BUILD
-	if (!use_bmem) {
-		char		*root;
-		PMEMoid		 root_oid;
-		PMEMobjpool *pop = (PMEMobjpool *)umm->umm_pool->up_priv;
+	switch (umm->umm_id) {
+		PMEMobjpool		*pop;
+		char			*root;
+		PMEMoid			 root_oid;
+		dav_obj_t		*dav_pop;
+		struct ad_blob_handle	 bh;
+
+	case UMEM_CLASS_PMEM:
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment